### PR TITLE
UPDATE: Documentation glossary and fixes

### PR DESCRIPTION
**Pull request by kosch104**

### CODE REVIEW

**Comment by Segelzwerg** [comment]
Did you forget to push something? There is nothing about the glossary?

File: doc/documentation/10_glossary.adoc [hunk 1]
```diff
@@ -12,15 +12,28 @@ Ist auch dafür nützlich ein einheitliches Verständnis zu schaffen und Diskuss
...
-|DirectoryPermissions
-|Die Berechtigungen (Lesen, Hochladen, Löschen) die jede Rolle hat; einem Ordner zugeordnet
+|Root Directory
+|Ein Ordner, der eine Gruppe im System repräsentiert. Jede Gruppe hat genau ein Root Directory.
+
+|Directory Permissions
+|Die Berechtigungen (Lesen, Hochladen, Löschen) die jede Rolle hat; einem Ordner zugeordnet.
+
+|Test (Modultest)
+|Modultests überprüfen dauerhaft die Funktion alle Komponenten (Module) des Systems. Dadurch wird sichergestellt, dass
```
Comment (Segelzwerg) [inline doc/documentation/10_glossary.adoc]:
```suggestion
|Modultests überprüfen dauerhaft die Funktion aller Komponenten (Module) des Systems. Dadurch wird sichergestellt, dass
```

File: doc/documentation/08_design_decisions.adoc [hunk 2]
```diff
@@ -100,7 +100,7 @@ Packages kommunizieren kann.
 
 Der dritte Test überprüft, ob die Schicht-Architektur eingehalten wird.
 Die zusätzliche Annotation ermöglicht es, die `AggregateRoot` zu markieren, um die Überprüfung zu erleichtern. Die
-Lesbarkeit und Verständlichkeit der Klassen und Packages wird dadurch auch verbessert. Hier haben wir am 09.03. drei
+Lesbarkeit und Verständlichkeit der Klassen und Packages wird dadurch auch verbessert. Hier haben wir drei
```
Comment (KaFe23) [inline doc/documentation/08_design_decisions.adoc]:
```suggestion
Lesbarkeit und Verständlichkeit der Klassen und Packages werden dadurch auch verbessert. Hier haben wir drei
```
Does this sound better?

Comment (kosch104) [reply doc/documentation/08_design_decisions.adoc]: I think it's wrong - "Lesbarkeit/Verständlichkeit wird verbessert", not "Lesbarkeit/Verständlichkeit werden verbessert"

File: doc/documentation/04_solution_strategy.adoc [hunk 3]
```diff
@@ -53,9 +56,16 @@ Dies soll es einfach im Log machen den `Stacktrace` nach zu verfolgen.
 Als Technologie benutzen wir `SLF4J` (Simple Logging Face for Java).
 Dies ist nativ in `Spring-Boot 2.x` und lässt sich mit einer einfachen Annotation (`SLF4J`) jeder Klasse hinzufügen.
 
+Der Großteil unserer Erfahrungen beruhte bisher auf MySQL. Da MariaDB die aktive Open-Source-Weiterentwicklung von MySQL
+ist, haben wir uns zunächst für die Verwendung von MariaDB entschieden.
+
```
Comment (iTitus) [inline doc/documentation/04_solution_strategy.adoc]:
```suggestion
```

File: doc/documentation/04_solution_strategy.adoc [hunk 4]
```diff
@@ -53,9 +56,16 @@ Dies soll es einfach im Log machen den `Stacktrace` nach zu verfolgen.
...
 Wir haben `SLF4J` nicht konfiguriert, also wird standardmäßig `Logback` benutzt.
 
 Wir haben uns dafür entschieden, weil unsere Internetrecherche die meisten Treffer dazu ergeben hat und unser Tutor uns unabhängig davon zu geraten hat.
 Die Einfachheit des Tools ist ebenfalls ein Entscheidungsgrund gewesen.
 
+Leider dauert das Starten der MariaDB-Datenbank in einem Docker-Container recht lange (ähnlich wie MySQL) und es ist
+umständlich, diesen Docker-Container dauerhaft aktiviert zu haben. Aus diesem Grund haben wir uns entschieden , während
+der Entwicklung und Testing H2 zu verwenden. H2 ist eine Embedded In-Memory Datenbank, die sehr schnell startet und
+die wir bereits kennengelernt haben.
```
Comment (iTitus) [inline doc/documentation/04_solution_strategy.adoc]:
```suggestion
```

File: doc/documentation/03_system_scope_and_context.adoc [hunk 5]
```diff
@@ -41,6 +41,6 @@ Diese verschiedenen Anfragen sind notwendig für die verschiedenen Funktionalit
 - Überprüfung von Berechtigungen eines Users
 - Erstellung einer Platzhalterberechtigung für alle Rollen
 
-Da uns während der Entwicklungszeit der externe Service nicht zur Verfügung stand, haben wir einen Stub entwickelt, welcher fixe Rollen zurückgibt.
-Wir haben uns dazu an dem REST-API Wiki der `Gruppenfindung` orientiert.
+Da uns während der Entwicklungszeit der externe Service nicht zur Verfügung stand, haben wir einen Stub entwickelt,
+festgesetzte fixe Rollen zurückgibt. Wir haben uns dazu an dem REST-API Wiki der `Gruppenfindung` orientiert.
```
Comment (iTitus) [inline doc/documentation/03_system_scope_and_context.adoc]:
```suggestion
der festgesetzte Rollen zurückgibt. Wir haben uns dazu an dem REST-API Wiki der `Gruppenfindung` orientiert.
```

**Pull request merged**